src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.BlockAnalyzer.cs

```diff
@@ -312,6 +319,14 @@ private bool ShouldAnalyze(IOperation operationBlock, ISymbol owningSymbol)
                         return false;
                     }
 
+                    //  7. Bail out on invalid operations, i.e. code with semantic errors.
+                    //     We are likely to have false positives from flow analysis results
+                    //     as we will not account for potential lambda/local function invocations.
+                    if (_hasInvalidOperation)
+                    {
+                        return false;
+                    }
+
                     //  7. Otherwise, we execute analysis by walking the reaching symbol write chain to attempt to
```

Comment:
need to renumber.

Reply:
The numbering got corrected in subsequent commits, verified current numbering is fine.